install-deps.sh

```diff
@@ -57,6 +57,7 @@ if [ x`uname`x = xFreeBSDx ]; then
         www/py-flask \
         www/fcgi \
         sysutils/flock \
+	sysutils/fusefs-libs \
```

Comment:
nit, could you use spaces instead of tab? we don't have the modelines for emacs/vim in this file, so it would look unaligned this way with different default settings than tabstop=8.